### DISCUSSION TITLE
希望在Waline评论区中能够显示Windows11标识

### DISCUSSION THREAD

**Discussion by Moemu** [walinejs/waline#656] [Ideas]
如标题，现在越来越多的用户选择升级Windows11，但他们在Waline评论区中仍然会显示Windows10标识
![](https://i.bmp.ovh/imgs/2021/11/820599d8f0eda17c.png)

**Comment by Mister-Hope** [comment]
你大可把这个锅丢给微软，微软的win11其实更像换皮win10，其内核版本号还是10

**Comment by Mister-Hope** [comment]
see here https://docs.microsoft.com/en-us/microsoft-edge/web-platform/how-to-detect-win11 chrome94 or less even can not detect

**Reply by Moemu** [reply]
感谢您提供的链接，事实上，我认为我们可以对支持区分Windows 11的浏览器做个适配，与此同时，对其他浏览器的Windows版本显示可以使用`Windows 10+`以代替
（开发者们若选择不支持也没有关系，我尊重开发者的意愿，只是评论时看起来有点怪）

**Reply by Mister-Hope** [reply]
想区分到是能做，回头看看把，先扔在这。这个需要前端配合。

**Comment by lizheming** [comment]
这个 Waline 已经支持了

---
This Waline already supports
